docs/_docs/snapshots/snapshots.adoc

```diff
@@ -178,6 +178,51 @@ the topology and wait while the data gets rebalanced and indexes are rebuilt.
 |N > M | Unsupported.
 |===
 
+=== Automatic snapshot restore procedure
+The automatic restore procedure allows the user to restore cache group(s) from a snapshot on an active cluster using the Java API or Command Language Interface (CLI).
+
+Currently, this procedure has a number of limitations:
+
+* The restore procedure applies only to public cache groups.
+* Cache groups to be restored from the snapshot must not present in the cluster, if they present, they must be destroyed by the user before starting this operation.
```

Comment:
Cache groups to be restored from the snapshot must not be present in the cluster. If they are present, they must be destroyed by the user before starting this operation.